chia/plotters/bladebit.py

```diff
@@ -147,10 +183,13 @@ def get_bladebit_install_info(plotters_root_path: Path) -> Optional[Dict[str, An
         if memory_warning is not None:
             info["bladebit_memory_warning"] = memory_warning
 
+    info["cuda_support"] = cuda_available
+
     return info
 
 
-progress_bladebit_ram = {
+# @TODO Set valid progress logs
```

Comment:
what does this refer to?

Reply:
Actually this part is the expected log output from cudaplot.
Currently I cannot confirm cudaplot log output, so I just put a todo message here.
After bladebit development finishes, I'll update this part.